db/migrate/20181101113907_create_comments.rb

```diff
@@ -0,0 +1,14 @@
+class CreateComments < ActiveRecord::Migration[5.2]
+  def change
+    create_table :comments do |t|
+      t.integer :commentable_id
+      t.string :commentable_type
+      t.text :content
+      t.references :user, foreign_key: true
+
+      t.timestamps
+    end
+
+    add_index :comments, [:commentable_type, :commentable_id]
```

Comment:
was this auto generated or you added it?

Reply:
@taimur-akhtar I added it.